lib/seattleflu/db/cli/command/clinical.py

```diff
@@ -230,12 +233,15 @@ def insert_clinical(df: pd.DataFrame, cursor):
 def print_problem_barcodes(problem_barcodes: pd.DataFrame, output: str):
     """
     Given a pandas DataFrame of *problem_barcodes*, writes the data to
-    stdout unless a filename *output* is given.
+    the log unless a filename *output* is given.
     """
     if output:
         problem_barcodes.to_csv(output, index=False)
     else:
-        print(problem_barcodes.to_csv(index=False))
+        problem_barcodes.apply(lambda x: LOG.warning(
+            f"{x['problem']} in MRN {x['MRN']}, Accession {x['Accession']}, "
+            f"barcode {x['barcode']}"
+        ), axis=1)
```

Comment:
This was my fault for suggesting it, but I realize now that we probably don't want to be logging PHI as it'll end up in the system logs. System logs may end up being shipped off-machine too, depending on the system. :grimacing: Can we instead log the problem, barcode (if any), filename, sheet, and row number? I think that will let someone (one of us!) track down problems without revealing any PHI. Sending full details with PHI to the optional CSV output is still fine, as it's obvious to the end user where that data goes.

Reply:
This has been fixed in the latest push

Reply:
Commit 553e8cf2cd33b2901b97ef7041f107f15486d5fe

Reply:
Thank you! Sorry for suggesting the wrong thing.